example/src/main/java/io/netty/example/http2/helloworld/server/Http2Server.java

```diff
@@ -69,12 +69,11 @@ public static void main(String[] args) throws Exception {
             sslCtx = null;
         }
         // Configure the server.
-        EventLoopGroup bossGroup = new NioEventLoopGroup(1);
-        EventLoopGroup workerGroup = new NioEventLoopGroup();
+        EventLoopGroup group = new NioEventLoopGroup();
         try {
             ServerBootstrap b = new ServerBootstrap();
             b.option(ChannelOption.SO_BACKLOG, 1024);
-            b.group(bossGroup, workerGroup)
+            b.group(group)
```

Comment:
@normanmaurer @Scottmitch I actually have a question here. When I share the event loop group with the boss and worker group I will have an event loop that will be both busy handling events for different channels AND accepting new channels. Wouldn't it be preferable to dedicate an event loop to the boss group, thus creating an event loop group with a single event loop for it?